test/all_of_ref_schema_test.rb

```diff
@@ -21,15 +21,15 @@ def test_all_of_ref_subschema_errors
     errors = JSON::Validator.fully_validate(schema, data, :errors_as_objects => true)
     nested_errors = errors[0][:errors]
     assert_equal([:allof_0], nested_errors.keys, 'should have nested errors for each allOf subschema')
-    assert_match(/the property '#\/name' of type String did not match the following type: integer/i, nested_errors[:allof_0][0][:message])
+    assert_match(/the property '#\/name' of type string did not match the following type: integer/i, nested_errors[:allof_0][0][:message])
```

Comment:
Can we please remove these text changes, as they are not related to the change to ruby 2.4